book/specifications/kimchi/template.md

```diff
@@ -213,6 +213,12 @@ Kimchi currently supports two lookup tables:
 
 {sections.table_12bit}
 
+##### Runtime tables
+
+Another type of lookup tables has been suggested in the [Extended Lookup tables RFC](../rfcs/extended-lookup-tables.md).
```

Comment:
Could you please change it to `[Extended Lookup Tables](../kimchi/extended-lookup-tables.md)`?

Also do we have some mechanism to check broken references? Like https://docs.rs/mdbook-linkcheck/latest/mdbook_linkcheck/

Reply:
Feel free to add mdbook linkcheck. Would be nice.

Reply:
It is already in the Makefile, just not run or fully configured.

Reply:
Opened follow-up issue: https://github.com/o1-labs/proof-systems/issues/1359